inc/cookie-policy.php

```diff
@@ -0,0 +1,101 @@
+		'<li>' . __( 'Right of access: You have the right to access your personal data that is known to us.', 'altis-consent' ) . '</li>' .
+		'<li>' . __( 'Right to rectification: you have the right to supplement, correct, have deleted or blocked your personal data whenever you wish.', 'altis-consent' ) . '</li>' .
+		'<li>' . __( 'If you give us your consent to process your data, you have the right to revoke that consent and to have your personal data deleted.', 'altis-consent' ) . '</li>' .
+		'<li>' . __( 'Right to transfer your data: you have the right to request all your personal data from the controller and transfer it in its entirety to another controller.', 'altis-consent' ) . '</li>' .
+		'<li>' . __( 'Right to object: you may object to the processing of your data. We comply with this, unless there are justified grounds for processing.', 'altis-consent' ) . '</li>' .
+	'</ul>';
+	$strings[] = '<p>' . __( 'To exercise these rights, please contact us. Please refer to the contact details at the bottom of this Cookie Policy. If you have a complaint about how we handle your data, we would like to hear from you, but you also have the right to submit a complaint to the supervisory authority (the Data Protection Authority).', 'altis-consent') . '</p>';
+	$strings[] = '<h3>' . __( 'Enabling, disabling and deleting cookies', 'altis-consent' ) . '</h3>';
+	$strings[] = '<p>' . __( 'You can use your internet browser to automatically or manually delete cookies. You can also specify that certain cookies may not be placed. Another option is to change the settings of your internet browser so that you receive a message each time a cookie is placed. For more information about these options, please refer to the instructions in the Help section of your browser.', 'altis-consent' ) . '</p>';
+	$strings[] = '<p>' . __( 'Please note that our website may not work properly if all cookies are disabled. If you do delete the cookies in your browser, they will be placed again after your consent when you visit our websites again.', 'altis-consent' ) . '</p>';
+	$strings[] = '<h2>' . __( 'Contact details', 'altis-consent' ) . '</h2>';
+	$strings[] = '<p>' . __( 'For questions and/or comments about our Cookie Policy and this statement, please contact us by using the following contact details:', 'altis-consent' ) . '</p>';
+	$strings[] = '<address>' . __( '[Organization Name]', 'altis-consent' ) . '<br />' .
+		__( '[Company Address]', 'altis-consent' ) . '<br />' .
+		__( '[Company Country]', 'altis-consent' ) . '<br />' .
+		sprintf( __( 'Website: %s', 'altis-consent' ), $domain ) . '<br />' .
+		sprintf( __( 'Email: %s', 'altis-consent' ), get_option( 'admin_email' ) ) . '<br />' .
+		__( '[Company Telephone]', 'altis-consent' ) .
+	'</address>';
+
+	if ( $blocks ) {
+		foreach ( $strings as $key => $string ) {
+			if ( 0 === strpos( $string, '<p>' ) ) {
```

Comment:
It's a shame that duplicate keys aren't supported! Would be a handy approach.

You could do something like:

```
$strings[] = [
    'tag' => 'h2',
    'text' => __( '...' ),
];
```

And then `switch` on the tag value but I feel that may be overcomplicating things unnecessarily for something that won't change much. Interesting to think about anyway :)

Reply:
Yeah, I didn't think too hard about it. I looked at what Complianz did (which seems way overcomplicated for what we want) and ultimately based it off how the [Privacy Policy page content is handled in core](https://github.com/WordPress/WordPress/blob/master/wp-admin/options-privacy.php#L49-L79)

Reply:
Ah cool, sounds good to me